src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/MIReplicaReportingServiceTest.java

```diff
@@ -184,6 +184,87 @@ void should_save_task_and_get_task_from_reportable_task() {
                 });
     }
 
+    @ParameterizedTest
+    @CsvSource(value = {
+        "LEGAL_OPERATIONS,Legal Operations,PRIVATELAW,Private Law,PRLAPPS,Private Law,ASSIGNED,AutoAssign,Assigned",
+        "CTSC,CTSC,CIVIL,Civil,CIVIL,Civil,UNASSIGNED,Configure,Unassigned",
+        "JUDICIAL,Judicial,IA,Immigration and Asylum,Asylum,Asylum,ASSIGNED,AutoAssign,Assigned",
+        "ADMIN,Admin,PUBLICLAW,Public Law,PUBLICLAW,Public Law,UNASSIGNED,Configure,Unassigned",
+        ",,WA,WA,WaCaseType,WaCaseType,ASSIGNED,AutoAssign,Assigned",
+        "TEST,TEST,TEST,TEST,TEST,TEST,UNASSIGNED,Configure,Unassigned"
```

Comment:
Should we also show we handle blanks? That's a possibility that can come from the service teams